pkg/model/helpers.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+import (
+	"regexp"
+)
+
+var (
+	alertParameter = regexp.MustCompile(`^ModSecurity: Warning. Pattern match.+ (at|against) "?([^T][^X].+)"?\.$`)
+)
+
+// ExtractParameter extracts the paramter of an alert, if any
+func (alert ModsecAlert) ExtractParameter() (string, bool) {
```

Comment:
Is a value type intended here? switching to `alert *ModsecAlert` avoids future error searching

Reply:
Since I do not intend to modify the received `ModsecAlert` yes, value type is intended. What future error could that be?

Reply:
someone who does not see that immediately (or inexperienced):
"Why is my function not writing my change to the alert?"
After 1hr of debugging:
"It was a missing bloody asterisk...!"

Knowing myself here in similar situations

Reply:
Again, I don't intend this function to change the `ModsecAlert`.
Reading https://github.com/golang/go/wiki/CodeReviewComments#receiver-type I guess let's go with `when in doubt, use a pointer receiver`.